### ISSUE TITLE
Switching tab entries doesnt switch tab entry panels in Forui 0.5.0.

### ISSUE THREAD

**Issue by DanielM-Z** [forus-labs/forui#203]
**Describe the bug**
When I switch between the Tab entries only the first entry is shown. It doesnt switch to the second panel. This problem started appearing in version 0.5.0. In 0.4.0 it worked fine.

**To Reproduce**
The problem can be seen in the documentation. https://forui.dev/docs/tabs. If that should work in your browser create a new project and replace main.dart with the following: 
```
import 'package:flutter/material.dart';
import 'package:forui/forui.dart';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({super.key});

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
        builder: (context, child) => FTheme(
              data: FThemes.zinc.dark, //rose.light,
              child: child!,
            ),
        home: const Scaffold(
            body: FScaffold(
          content: TabsShowcase(),
        )));
  }
}

class TabsShowcase extends StatelessWidget {
  const TabsShowcase({super.key});

  @override
  Widget build(BuildContext context) {
    // This is copy pasted from https://forui.dev/docs/tabs.
    return FTabs(
      tabs: [
        FTabEntry(
          label: const Text('Account'),
          content: FCard(
            title: const Text('Account'),
            subtitle: const Text(
                'Make changes to your account here. Click save when you are done.'),
            child: Column(
              children: [
                const FTextField(
                  label: Text('Name'),
                  hint: 'John Renalo',
                ),
                const SizedBox(height: 10),
                const FTextField(
                  label: Text('Email'),
                  hint: 'john@doe.com',
                ),
                const SizedBox(height: 16),
                FButton(
                  label: const Text('Save'),
                  onPress: () {},
                ),
              ],
            ),
          ),
        ),
        FTabEntry(
          label: const Text('Password'),
          content: FCard(
            title: const Text('Password'),
            subtitle: const Text(
                'Change your password here. After saving, you will be logged out.'),
            child: Column(
              children: [
                const FTextField(label: Text('Current password')),
                const SizedBox(height: 10),
                const FTextField(label: Text('New password')),
                const SizedBox(height: 16),
                FButton(
                  label: const Text('Save'),
                  onPress: () {},
                ),
              ],
            ),
          ),
        ),
      ],
    );
  }
}

```

**Expected behavior**
The second panel should be displayed if I switch tabs.

**Device:**
 - Flutter version: 3.24.2
 - Forui version: 0.5.0
 - Device: Desktop.
 - OS: Windows 11 Home
 - AMD Ryzen 5 5600X 6-Core Processor

**Additional context**
Im using Flutter Rust Bridge which might mess things up.

**Comment by Pante** [comment]
Thanks for taking the time to report this issue, it seems to have been introduced in 3d66d2d0dc37fcb0cc2342b2f849d849543cbd20. It's a pretty bad bug, we'll probably publish a patch for it tomorrow.

**Comment by Pante** [comment]
We've released `0.5.1` which should fix this issue. Docs will be updated at a later date.
